[Bug Report] npm run dist

<!-- generated by https://eleme-issue.surge.sh DO NOT REMOVE -->

### Element UI version
2.1.0

### OS/Browsers version
windows10

### Vue version
2.5.13

### Reproduction Link
[https://github.com/ElemeFE/element](https://github.com/ElemeFE/element)

### Steps to reproduce
npm run dist 

### What is Expected?
create dist 

### What is actually happening?
> eslint src/**/* test/**/* packages/**/* build/**/* --quiet


C:\Users\pc\Desktop\element-dev\packages\table\src\table.vue
  539:14  error  Missing semicolon  semi

C:\Users\pc\Desktop\element-dev\packages\theme-chalk\src\tabs.scss
  1:1  error  Parsing error: Unexpected character '@'

C:\Users\pc\Desktop\element-dev\packages\theme-chalk\src\tag.scss
  1:1  error  Parsing error: Unexpected character '@'

C:\Users\pc\Desktop\element-dev\packages\theme-chalk\src\time-picker.scss
  1:1  error  Parsing error: Unexpected character '@'

C:\Users\pc\Desktop\element-dev\packages\theme-chalk\src\time-select.scss
  1:1  error  Parsing error: Unexpected character '@'

C:\Users\pc\Desktop\element-dev\packages\theme-chalk\src\tooltip.scss
  1:1  error  Parsing error: Unexpected character '@'

C:\Users\pc\Desktop\element-dev\packages\theme-chalk\src\transfer.scss
  1:1  error  Parsing error: Unexpected character '@'

C:\Users\pc\Desktop\element-dev\packages\theme-chalk\src\tree.scss
  1:1  error  Parsing error: Unexpected character '@'

C:\Users\pc\Desktop\element-dev\packages\theme-chalk\src\upload.scss
  1:1  error  Parsing error: Unexpected character '@'

✖ 84 problems (84 errors, 0 warnings)
  1 error, 0 warnings potentially fixable with the `--fix` option.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! element-ui@2.1.0 lint: `eslint src/**/* test/**/* packages/**/* build/**/* --quiet`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the element-ui@2.1.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\pc\AppData\Roaming\npm-cache\_logs\2018-02-06T07_38_45_540Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! element-ui@2.1.0 dist: `npm run clean && npm run build:file && npm run lint && cooking build -c build/cooking.conf.js,build/cooking.common.js,build/cooking.component.js -p && npm run build:utils && npm run build:umd && npm run build:theme`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the element-ui@2.1.0 dist script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\pc\AppData\Roaming\npm-cache\_logs\2018-02-06T07_38_45_591Z-debug.log

<!-- generated by https://eleme-issue.surge.sh DO NOT REMOVE -->

--- a/.eslintignore
+++ b/.eslintignore
@@ -6,3 +6,4 @@ node_modules
 lib
 coverage
 *.md
+*.scss

--- a/packages/table/src/table.vue
+++ b/packages/table/src/table.vue
@@ -536,7 +536,7 @@
           if (this.showSummary) {
             return {
               height: this.layout.tableHeight ? this.layout.tableHeight + 'px' : ''
-            }
+            };
           }
           return {
             height: this.layout.viewportHeight ? this.layout.viewportHeight + 'px' : ''

